nestkernel/conn_builder.cpp

```diff
@@ -351,19 +343,19 @@ nest::ConnBuilder::single_connect_( index snode_id, Node& target, thread target_
 
   for ( size_t indx = 0; indx < synapse_model_id_.size(); ++indx )
   {
-    DictionaryDatum param_dict = create_param_dict_( snode_id, target, target_thread, rng, indx );
+    create_param_dict_( snode_id, target, target_thread, rng, indx );
 
     if ( default_weight_and_delay_[ indx ] )
     {
-      kernel().connection_manager.connect( snode_id, &target, target_thread, synapse_model_id_[ indx ], param_dict );
+      kernel().connection_manager.connect( snode_id, &target, target_thread, synapse_model_id_[ indx ], dummy_param_dicts_[ target_thread ] );
```

Comment:
It seems confusing that all `connect()` calls pass a `dummy_...` argument. We should think more about names here.